### PR TITLE
add json-e 3.0.0

### PR DESCRIPTION
**Pull request by bollwyvl**
Checklist

- [*] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [*] Source is from official source
- [*] Package does not vend other packages
- [*] Build number is 0
- [*] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [*] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by bollwyvl** [comment]
@conda-forge/help-python this appears to be in working order, ready for review...

File: recipes/json-e/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: d2914f785d93ecc4f0b2ad6e3f2791f33327eaa740a3c4917d68a9a485dd282d
+    folder: dist
+  - url: https://github.com/taskcluster/{{ name }}/archive/v{{ version }}.tar.gz
```
Comment (marcelotrevisani) [inline recipes/json-e/meta.yaml]: I think you just need the ``github`` tarballs right?
You can delete the package being download from pypi and install everything from the tarballs

**Comment by bollwyvl** [comment]
Fair enough!

**Comment by bollwyvl** [comment]
Eek, looks like it grabs some extra files from the source tarball... having a look...

File: recipes/json-e/meta.yaml (hunk 1 above)
Comment (bollwyvl) [reply recipes/json-e/meta.yaml]: Yep, building/installing the sdist works fine.

**Comment by bollwyvl** [comment]
thanks!

**Pull request merged**